app/Console/Commands/CreateMasterVoucherLogReport.php

```diff
@@ -0,0 +1,308 @@
+<?php
+
+namespace App\Console\Commands;
+
+use Carbon\Carbon;
+use DB;
+use Excel;
+use Illuminate\Support\Facades\Storage;
+use Illuminate\Console\Command;
+use Log;
+use Maatwebsite\Excel\Writers\LaravelExcelWriter;
+use ZipArchive;
+
+class CreateMasterVoucherLogReport extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'arc:createMVLReport 
+                            {--force : Execute without confirmation, eg for automation}
+                            {--no-zip : Don\'t wrap files in a single archive}
+                            {--plain : Don\'t Encrypt contents of files}
+                            '
+    ;
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Creates, encrypts and stores the MVL report under in /storage';
+
+    /**
+     * The default disk we want.
+     *
+     * @var string $disk
+     */
+    private $disk = 'enc';
+
+    /**
+     * The default archive name.
+     *
+     * @var string $archiveName
+     */
+    private $archiveName = "MVLReport.zip";
+
+    /**
+     * The sheet headers.
+     *
+     * @var array $headers
+     */
+    private $headers = [
+        'Area',
+        'Voucher Code',
+        'Dispatch Date',
+        'Disbursed Date',
+        'RVID',
+        'Participant',
+        'Payment Request Date',
+        'Trader Name',
+        'Retail Outlet',
+        'Reimbursed Date'
+    ];
+
+    /**
+     * The report's query template
+     *
+     * TODO: refactor this as eloquent lookups when it's finalised.
+     *      This will eventually help with the fact we'll need to chunk data too.
+     * @var string $report
+     */
+    private $report = <<<EOD
+SELECT
+  sponsors.name AS 'Area',
+  vouchers.code AS 'Voucher Code',
+  dispatch_date AS 'Dispatch Date',
+  disbursed_at AS 'Disbursed Date',
+  rvid AS 'RVID',
+  pri_carer_name AS 'Participant',
+  payment_request_date AS 'Payment Request Date',
+  trader_name AS 'Trader Name',
+  market_name AS 'Retail Outlet',
```

Comment:
lol 'Retail Outlet'

Reply:
Personally i'd consider the trader stall that, rather than the market as a whole, but we'll use ARC's terms.